Bottom: e77b7f77d7fbad34159e8267088d5ef21fd2a1ba
Top:    b1fb469d2de9299a0f18930277dbd0e349c59c70
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-17 01:58:53 +0000

Make --enable-gl default when building the GTK HID


---

diff --git a/configure.ac b/configure.ac
index 230b8e5..c452997 100644
--- a/configure.ac
+++ b/configure.ac
@@ -392,8 +392,12 @@ fi
 AC_MSG_CHECKING([for whether to use GL drawing])
 AC_ARG_ENABLE([gl],
 [  --enable-gl           Enable GL drawing (with GTK HID)],
-[],[enable_gl=no])
-
+[],[
+	case " $with_gui " in
+		*\ gtk\ *) enable_gl=yes;;
+		* ) enable_gl=no;;
+	esac
+])
 AC_MSG_RESULT([$enable_gl])
 AM_CONDITIONAL(USE_GL, test x$enable_gl = xyes)
